plugins/polio/budget/serializers.py

```diff
@@ -79,13 +82,34 @@ class AvailableRoundsSerializer(serializers.Serializer):
     budget_process_id = serializers.IntegerField()
 
 
+class BudgetProcessNestedRoundSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Round
+        fields = [
+            "id",
+            "number",
+            "cost",
+            "target_population",
+        ]
+
+
+class BudgetProcessUWriteRoundSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Round
+        fields = [
+            "id",
+            "cost",
```

Comment:
It's terrifying how the addition of a _single_ field in a nested writable serializer can generate so much extra code in the parent validation logic…

There should be a simpler solution, but I can't find it…
So I've got nothing better to suggest.